packages/server/lib/browsers/chrome.ts

```diff
@@ -294,6 +294,13 @@ const _navigateUsingCRI = async function (client, url) {
   await client.send('Page.navigate', { url })
 }
 
+const _setDownloadsDir = async function (client, dir) {
+  await client.send('Page.setDownloadBehavior', {
+    behavior: 'allow',
+    downloadPath: dir,
+  })
+}
```

Comment:
if you wanted, you could deduplicate this with electron.js by moving this to a function on `cri-client` like `client.setDownloadsDir(dir)`

Reply:
Since electron.js doesn't use `cri-cient` and even the method name used (`send` vs `sendCommand`) is different, the best I could do is abstract out the command name string and the two-property object. I'm not sure it's worth the indirection.